refinery/data_set_manager/tests.py

```diff
@@ -2291,6 +2292,11 @@ def test_post_good_isa_tab_url(self, delay_mock):
         self.successful_import_assertions()
 
 
+# This override_settings allows for test-data/ fixtures to be copied into
+# Refinery's file_store on import rather than being moved in Travis CI test
+# runs
```

Comment:
Grammar: add "."

Reply:
And I might be out of the loop, but the explanation isn't really clear to me... this isn't python `import`, right? And it's a copy in either case, right?